requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+-r requirements.txt
+mock
+pep8==1.6.2
```

Comment:
How about a less strict `>=1.6.0`? The behaviour for E126 was changed in version 1.6.0 already: https://github.com/jcrocholl/pep8/blob/master/CHANGES.txt#L68
Of course, if we want to stay in perfect sync with Travis, we should probably require the latest released version and update the requirements file whenever a new pep8 version is released. Not sure what the best strategy is, you seem to be more versed in this.